Formula/sundials.rb

```diff
@@ -0,0 +1,54 @@
+class Sundials < Formula
+  desc "Nonlinear and differential/algebraic equations solver"
+  homepage "https://computation.llnl.gov/casc/sundials/main.html"
+  url "https://computation.llnl.gov/projects/sundials/download/sundials-2.7.0.tar.gz"
+  sha256 "d39fcac7175d701398e4eb209f7e92a5b30a78358d4a0c0fcc23db23c11ba104"
+  revision 2
+
+  option "with-openmp", "Enable OpenMP multithreading"
+
+  depends_on "cmake" => :build
+  depends_on :fortran => :recommended
+  depends_on :mpi => [:cc, :f77, :recommended]
+  depends_on "suite-sparse" => :recommended
+  depends_on "openblas" => :optional
```

Comment:
This option should be removed because we're no longer going to support having `openblas` options.